src/pages/manager/OpenAccountPage.js

```
@@ -1,13 +1,38 @@
 import { expect } from '@playwright/test';
 
-export class AddCustomerPage {
+export class OpenAccountPage {
   constructor(page) {
     this.page = page;
+    this.selectCustomerLocator = page.locator('select[name="userSelect"]');
+    this.selectCurrencyLocator = page.locator('select[name="currency"]');
   }
 
   async open() {
-    await this.page.goto(
-      '/angularJs-protractor/BankingProject/#/manager/openAccount',
-    );
+    await this.page.goto('#/manager/openAccount');
+  }
+
+  async selectCustomerOption(fullName) {
+    await this.selectCustomerLocator.selectOption({ label: fullName });
```

Selecting a customer: `await this.selectCustomerLocator.selectOption({ label: fullName });` will silently succeed with no selection if the provided label does not exist. Consider validating the option exists before selecting (e.g., check `this.selectCustomerLocator.locator('option', { hasText: fullName })` count > 0) or wait for the dropdown to be populated before calling `selectOption`. This prevents subtle test failures when the test setup hasn't created the expected customer yet.